### ISSUE TITLE
"length" as a key is producing parsing errors

### ISSUE THREAD

**Issue by Nic128** [mrodrig/json-2-csv#49]
I was creating a json based on the joi language file: https://github.com/hapijs/joi/blob/master/lib/language.js
````
{
  "validation": {
    "joi": {
      "any": {
        "unknown": "is not allowed",
        "invalid": "contains an invalid value",
        "empty": "is not allowed to be empty",
        "required": "is required",
        "allowOnly": "must be one of {{valids}}",
        "default": "threw an error when running default method"
      },
      "alternatives": {
        "base": "not matching any of the allowed alternatives"
      },
      "array": {
        "length": "must contain {{limit}} items"
      },
      "boolean": {
        "base": "must be a boolean"
      },
      "binary": {
        "base": "must be a buffer or a string",
        "min": "must be at least {{limit}} bytes",
        "max": "must be less than or equal to {{limit}} bytes",
        "length": "must be {{limit}} bytes"
      }
    }
  }
}

````

Trying to parse it to csv I was missing some headers, turns out having "length" as a key breaks the conversion:
````
validation.joi.any.unknown,validation.joi.any.invalid,validation.joi.any.empty,validation.joi.any.required,validation.joi.any.allowOnly,validation.joi.any.default,validation.joi.alternatives.base,validation.joi.array,validation.joi.boolean.base,validation.joi.binary
is not allowed,contains an invalid value,is not allowed to be empty,is required,must be one of {{valids}},threw an error when running default method,not matching any of the allowed alternatives,must contain {{limit}} items,must be a boolean,must be a buffer or a string,must be at least {{limit}} bytes,must be less than or equal to {{limit}} bytes,must be {{limit}} bytes
````

**Comment by mrodrig** [comment]
Thanks for reporting this @Nic128.  I'm looking into it now.

**Comment by mrodrig** [comment]
This should be fixed in version 2.0.6 on NPM now.  Thanks again for reporting this and sorry for the delay.

**Comment by Nic128** [comment]
It works. Thanks.
